addOns/reports/src/main/zapHomeFiles/reports/risk-confidence-html/report.html

```diff
@@ -0,0 +1,387 @@
+							</ol>
+						</li>
+						<li data-th-if="${reportData.isIncludeSection('alerts')}"><a href="#alerts" data-th-text="#{report.template.section.alerts}">Alerts</a>
+							<ol>
+								<th-block data-th-each="riskCode: ${includedRiskCodes}" data-th-with="filteredAlerts=${#riskCode=riskCode, alerts.{? #this.userObject.risk == #riskCode}}">
+									<th-block data-th-unless="${filteredAlerts.isEmpty()}">
+										<th-block data-th-each="confidenceCode: ${includedConfidenceCodes}" data-th-with="filteredAlerts=${#confidenceCode=confidenceCode, filteredAlerts.{? #this.userObject.confidence == #confidenceCode}}">
+											<li data-th-unless="${filteredAlerts.isEmpty()}"><a data-th-href="@{#alerts--risk-{riskCode}-confidence-{confidenceCode}(riskCode=${riskCode}, confidenceCode=${confidenceCode})}"><span data-th-text="#{report.template.risk}">Risk</span>=<span class="risk-level" data-th-text="${helper.getRiskString(riskCode)}">Risk level</span>, <span data-th-text="#{report.template.confidence}">Confidence</span>=<span class="confidence-level" data-th-text="${helper.getConfidenceString(confidenceCode)}">Confidence level</span> <span data-th-text="|(${filteredAlerts.size()})|">(count)</span></a></li>
+										</th-block>
+									</th-block>
+								</th-block>
+							</ol>
+						</li>
+						<li data-th-if="${reportData.isIncludeSection('appendix')}"><a href="#appendix" data-th-text="#{report.template.section.appendix}">Appendix</a>
+							<ol>
+								<li data-th-if="${reportData.isIncludeSection('alertTypes')}"><a href="#alert-types" data-th-text="#{report.template.section.alertTypes}">Alert types</a></li>
+							</ol>
+						</li>
+					</data-th-block>
+				</ol>
+			</nav>
+		</section>
+
+		<section data-th-if="${reportData.isIncludeSection('aboutThisReport')}" id="about-this-report" class="about-this-report">
+			<h2 data-th-text="#{report.template.section.aboutThisReport}">About this report</h2>
+
+			<section data-th-if="${reportData.isIncludeSection('reportDescription')}" id="report-description">
+				<h3 data-th-text="#{report.template.section.reportDescription}">Report description</h3>
+				<div class="report-description--container">
+					<th-block data-th-replace="::nl2pbr(text=${description})">
+						<p>Report description</p>
+					</th-block>
+				</div>
+			</section>
+
+			<section data-th-if="${reportData.isIncludeSection('reportParameters')}" id="report-parameters">
+				<h3 data-th-text="#{report.template.section.reportParameters}">Report parameters</h3>
+				<div class="report-parameters--container">
+					<h4 data-th-text="#{report.template.reportParameters.contexts}">Contexts</h4>
+					<th-block data-th-with="includedContexts=${reportData.getContexts()}">
+						<th-block data-th-switch="${includedContexts.size}">
+							<p data-th-case="0">No contexts were selected, so all contexts were included by default.</p>
```

Comment:
This and some of the other strings should ideally all be i18ned and therefore in the properties file.

Reply:
Fixed - thanks!